example/styles/stylesheet.css

```diff
@@ -271,7 +271,6 @@ a:hover {
 .albums {
   display: flex;
   justify-content: space-between;
-  flex-wrap: wrap;
```

Comment:
Let's keep both vertical and horizontal scroll in the example app? 

For example 8 cards per row or something like that: 
```css
.albums {
  display: flex;
  justify-content: space-between;
  flex-wrap: wrap;
  width: calc(150px * 10);
}
```